MSAL/src/native_auth/controllers/responses/SignUpResults.swift

```diff
@@ -72,18 +72,9 @@ enum SignUpVerifyCodeResult {
     case error(error: VerifyCodeError, newState: SignUpCodeRequiredState?)
 }
 
-enum SignUpResendCodeResult {
-    /// Returned if a user has received an email with code.
-    ///
-    /// - newState: An object representing the new state of the flow with follow on methods.
-    /// - sentTo: The email/phone number that the code was sent to.
-    /// - channelTargetType: The channel (email/phone) the code was sent through.
-    /// - codeLength: The length of the code required.
-    case codeRequired(newState: SignUpCodeRequiredState, sentTo: String, channelTargetType: MSALNativeAuthChannelType, codeLength: Int)
-
-    /// An error object indicating why the operation failed.
-    case error(ResendCodeError)
-}
+/// Result type that contains information about the code sent, the next state of the reset password process and possible errors.
```

Comment:
nit: these comments are here because we were planning to make these result public with the async/await introduction. Given that async/await isn't in the plan anymore, should we still keep updating this internal comments?

Reply:
Good point, I've removed them.